[2025-02-04T07:44:53.029+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-04T07:44:53.045+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: idp_fetch_mongo.fetch_from_mongo manual__2025-02-04T07:44:50.619597+00:00 [queued]>
[2025-02-04T07:44:53.051+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: idp_fetch_mongo.fetch_from_mongo manual__2025-02-04T07:44:50.619597+00:00 [queued]>
[2025-02-04T07:44:53.051+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 2
[2025-02-04T07:44:53.065+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): fetch_from_mongo> on 2025-02-04 07:44:50.619597+00:00
[2025-02-04T07:44:53.082+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=1499) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-02-04T07:44:53.083+0000] {standard_task_runner.py:72} INFO - Started process 1500 to run task
[2025-02-04T07:44:53.084+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'idp_fetch_mongo', 'fetch_from_mongo', 'manual__2025-02-04T07:44:50.619597+00:00', '--job-id', '162', '--raw', '--subdir', 'DAGS_FOLDER/idp.py', '--cfg-path', '/tmp/tmpfe2ufpe4']
[2025-02-04T07:44:53.086+0000] {standard_task_runner.py:105} INFO - Job 162: Subtask fetch_from_mongo
[2025-02-04T07:44:53.156+0000] {task_command.py:467} INFO - Running <TaskInstance: idp_fetch_mongo.fetch_from_mongo manual__2025-02-04T07:44:50.619597+00:00 [running]> on host 3aa1f743f988
[2025-02-04T07:44:53.242+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='idp_fetch_mongo' AIRFLOW_CTX_TASK_ID='fetch_from_mongo' AIRFLOW_CTX_EXECUTION_DATE='2025-02-04T07:44:50.619597+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-02-04T07:44:50.619597+00:00'
[2025-02-04T07:44:53.248+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2025-02-04T07:44:53.249+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2025-02-04T07:44:53.249+0000] {logging_mixin.py:190} INFO - Current task name:fetch_from_mongo state:running start_date:2025-02-04 07:44:53.045713+00:00
[2025-02-04T07:44:53.249+0000] {logging_mixin.py:190} INFO - Dag name:idp_fetch_mongo and current dag run status:running
[2025-02-04T07:44:53.250+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-02-04T07:44:53.265+0000] {base.py:84} INFO - Retrieving connection 'mongoid'
[2025-02-04T07:44:53.269+0000] {logging_mixin.py:190} INFO - Error fetching from MongoDB: MongoDB URI options are key=value pairs.
[2025-02-04T07:44:53.270+0000] {python.py:240} INFO - Done. Returned value was: []
[2025-02-04T07:44:53.299+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-02-04T07:44:53.300+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=idp_fetch_mongo, task_id=fetch_from_mongo, run_id=manual__2025-02-04T07:44:50.619597+00:00, execution_date=20250204T074450, start_date=20250204T074453, end_date=20250204T074453
[2025-02-04T07:44:53.317+0000] {logging_mixin.py:190} INFO - Task instance in success state
[2025-02-04T07:44:53.318+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: running
[2025-02-04T07:44:53.319+0000] {logging_mixin.py:190} INFO - Dag name:idp_fetch_mongo queued_at:2025-02-04 07:44:50.626677+00:00
[2025-02-04T07:44:53.319+0000] {logging_mixin.py:190} INFO - Task hostname:3aa1f743f988 operator:PythonOperator
[2025-02-04T07:44:53.349+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-02-04T07:44:53.367+0000] {taskinstance.py:3895} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-02-04T07:44:53.368+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
